src/components/button/index.md.njk

```diff
@@ -46,7 +46,7 @@ Avoid using multiple default buttons on a single page. Having more than one main
 
 ### Start buttons
 
-Use a start button for the main call to action on your service’s [start page](/patterns/start-pages/).
+Use a start button for the main call to action on your service’s [start page](/patterns/start-pages/). Start buttons are rendered as links rather than inputs, because they don't submit any data.
```

Comment:
not sure 'rendered' (drawn) is right here - they're drawn as buttons. Maybe 

> Start buttons actually use the <a> (link) element instead of <button>, because they don't submit any data. They look like buttons because they are the main call to action on the page.

Reply:
Thanks Joe - really good points.